apps/bfd-pipeline/bfd-pipeline-rda-bridge/run_bridge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+PROJ_DIR=$(dirname "$0")
+
+java -cp $PROJ_DIR/target/bfd-pipeline-rda-bridge-1.0.0-SNAPSHOT.jar:$PROJ_DIR/target/dependency/* gov.cms.bfd.pipeline.bridge.RDABridge "$@"
```

Comment:
shellcheck recommends double quoting the reference to $PROJ_DIR, or you could quote the whole cp argument.

also might be nice to use exec here like we do for the pipeline and application scripts

Reply:
I did try exec at first, but I had trouble initially getting the parameters to work with it so I switched to just a raw java execution.  I could look again to see if I can add exec as another option.